models/marts/fct_order_items.sql

```diff
@@ -33,17 +41,35 @@ final as (
     -- dimensions
         order_items.price,
         order_items.freight_value,
-        case when order_payments.payment_type is null then 'returned'
-            else orders.order_status end as order_status,
+        case 
+            when order_payments.payment_type is null then 'returned'
+            else orders.order_status 
+        end as order_status,
```

Comment:
this "returned" bit of logic is not immediately clear: what if it's yet to be paid and therefore null? A comment would be helpful

Reply:
agreed, not sure what my thought process was here tbh. There are 0 records with a null `order_status`
removed logic all together